docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/index.md

```diff
@@ -22,108 +22,90 @@ This is the signature of the constructor for the base Emitter class:
-| `port` | The port to connect to | No | Positive integer | `None` |
-| `method` | The method to use: “get” or “post” | No | String | `post` |
-| `buffer_size` | Number of events to store before flushing | No | Positive integer | `10` |
-| `on_success` | Callback executed when a flush is successful | No | Function taking 1 argument | `None` |
+| `protocol` | Request protocol: http or https | No | String | `https` | |
+| `port` | The port to connect to | No | Positive integer | `None` | |
+| `method` | The method to use: “get” or “post” | No | String | `post` | |
+| `batch_size` | Number of events to store before flushing | No | Positive integer | `10` | |
+| `on_success` | Callback executed when a flush is successful | No | Function taking 1 argument | `None` | |
 | `on_failure` | Callback executed when a flush is unsuccessful | No | Function taking 2 arguments | `None` |
 | `byte_limit` | Number of bytes to store before flushing | No | Positive integer | `None` |
-| `request_timeout` | Timeout for HTTP requests | No | Positive integer or tuple of 2 integers | `None` |
-
-- `protocol`
-
-`protocol` defaults to "https" but also supports "http".
-
-:::note Prior to v0.12.0
-
-In older versions, `protocol` defaulted to `http`.
-
+| `request_timeout` | Timeout for HTTP requests | No | Positive integer or tuple of 2 integers | `None` | v0.10.0 |
+| `max_retry_delay_seconds` | The maximum time between attempts to send failed events to the collector | No | int | 60s | v0.13.0 |
+| `buffer_capacity` | The maximum capacity of the event buffer | No | int | `None` | v0.13.0 |
+| `custom_retry_codes` | Custom retry rules for HTTP status codes received in emit responses from the Collector | No | dict | `None` | v0.13.0 |
+| `event_store` | Stores the event buffer and buffer capacity | No | EventStore | `None` | v0.13.0 |
+
+See the [`API docs`](https://snowplow.github.io/snowplow-python-tracker/) for more information on the individual parameters. 
+## What happens if an event fails to send?
+:::note New in v0.13.0
+Retry capabilities are new in v0.13.0
 :::
+After trying to send a batch of events the collector will return an http status code. A 2xx code is always considered successful. If a failure code is returned (anything other than 2xx, with certain exceptions, see below), the events (as PayloadDictList objects) are returned to the buffer. They will be retried in future sending attempts. 
 
-- `buffer_size`
-
-When the emitter receives an event, it adds it to a buffer. When the queue is full, all events in the queue get sent to the collector. The `buffer_size` argument allows you to customize the queue size. By default, it is 1 for GET requests and 10 for POST requests. (So in the case of GET requests, each event is fired as soon as the emitter receives it.) If the emitter is configured to send POST requests, then instead of sending one for every event in the buffer, it will send a single request containing all those events in JSON format.
```

Comment:
Was it intentional that you removed these docs about the `batch_size`, `byte_limit`, `on_success`, `request_timeout` and other properties? I thought they were quite well written, wondering if we can keep some of it.

Reply:
I removed these just because it seemed quite similar to the API docs. I'll add them back in, but I think it could be good to re-structure this page for V1 though?

Reply:
Feel free to move it around, I am just worried about losing the information. The API docs can be quite terse, but I think we can be more verbose in these docs and explain things in a more understandable way to users which I think that text did well.